test/cfg/googletest.cpp

```diff
@@ -71,7 +71,7 @@ TEST(Test, warning_in_assert_macros)
     ASSERT_GE(i, i);
 
     unsigned int u = errno;
-    // cppcheck-suppress unsignedPositive
+    // cppcheck-suppress [unsignedPositive, compareValueOutOfTypeRangeError]
```

Comment:
Interesting - I thought the format was `cppcheck-suppress id1;id2`.

Reply:
According to the manual: `// cppcheck-suppress [aaaa, bbbb]`